packages/tessellate-request/lib/configuration/oauth-configuration.js

```diff
@@ -0,0 +1,9 @@
+// @flow
+
+export type OAuthConfig = {
```

Comment:
If we're using a type in many different places, I would suggest putting them as declarations (https://flowtype.org/docs/declarations.html) into a separate file. E.g. under `packages/tessellate-request/flow/*.js`).